providers/flagd/README.md

```diff
@@ -177,6 +177,12 @@ By default, the provider is configured to
 use [least recently used (lru)](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/LRUMap.html)
 caching with up to 1000 entries.
 
+##### Context enrichment
```

Comment:
This can be used to select a subset of fields from the `getMetadata` response. By default, all of it is injected into the context.